components/Post/LatestPosts.tsx

```diff
@@ -17,11 +18,14 @@ const LatestPosts = ({ latestPosts }: Props) => {
         <div className="related-posts-block" key={post.title}>
           <Link href={post.slug}>
             <a className="image">
-              <img
+              <Image
                 src={post.featuredImage}
                 alt="blog post image"
-                width={102}
-                height={102}
+                layout="fill"
+                objectFit="cover"
+                width={800}
+                height={400}
+                quality={100}
```

Comment:
@ntorbinskiy can you send a screenshot of a difference between default value, and this value?

Reply:
![image](https://user-images.githubusercontent.com/60061661/195609032-7ac06844-b427-4d6a-83f4-51378037a4f8.png)
But in this situation- I don't need it

Reply:
![image](https://user-images.githubusercontent.com/60061661/195609032-7ac06844-b427-4d6a-83f4-51378037a4f8.png)
From [next.js docs](https://nextjs.org/docs/basic-features/image-optimization)
But in this situation

Reply:
@ntorbinskiy why do you send me the documentation?

I asked a question and the image quality difference between the default value which is 75, and the value is 100 which you specified